code/index.html

```diff
@@ -5,18 +5,111 @@
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
```

Comment:
Would be good to remove these comments. It could also be good to add your own comments to easily navigate through your code. It's something I should have done myself aswell!